bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -130,6 +144,13 @@ public Integer getSample() {
                                                   return pendingAddOps.size();
                                               }
                                           });
+
+        if (!metadata.isClosed() && !(this instanceof ReadOnlyLedgerHandle) && bk.getExplicitLacInterval() > 0) {
```

Comment:
can this be added to a protected function "initializeLacUpdateScheduler()", then in ReadOnlyLedgerHandle overwrite this method to do nothing?